configuration/set_test.go

```diff
@@ -0,0 +1,61 @@
+package configuration
```

Comment:
Is it worth adding a test to validate we do the default username/password check and reprompts??

Reply:
Added in: 10a38d3 AND fixed an issue with it, great suggestion Lord Zender!

Reply:
Added in: 10a38d3 AND fixed an issue with it, great suggestion Lord Zender!

Reply:
Added in: 10a38d3 AND fixed an issue with it, great suggestion Lord Zender!